server/app/views/applicant/NorthStarApplicantBaseView.java

```diff
@@ -0,0 +1,38 @@
+package views.applicant;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import controllers.AssetsFinder;
+import controllers.applicant.ApplicantRoutes;
+import modules.ThymeleafModule;
+import org.thymeleaf.TemplateEngine;
+import play.mvc.Http.Request;
+
+public abstract class NorthStarApplicantBaseView {
+  protected final TemplateEngine templateEngine;
+  protected final ThymeleafModule.PlayThymeleafContextFactory playThymeleafContextFactory;
+  protected final AssetsFinder assetsFinder;
+  protected final ApplicantRoutes applicantRoutes;
+
+  NorthStarApplicantBaseView(
+      TemplateEngine templateEngine,
+      ThymeleafModule.PlayThymeleafContextFactory playThymeleafContextFactory,
+      AssetsFinder assetsFinder,
+      ApplicantRoutes applicantRoutes) {
+    this.templateEngine = checkNotNull(templateEngine);
+    this.playThymeleafContextFactory = checkNotNull(playThymeleafContextFactory);
+    this.assetsFinder = checkNotNull(assetsFinder);
+    this.applicantRoutes = checkNotNull(applicantRoutes);
+  }
+
+  protected ThymeleafModule.PlayThymeleafContext createThymeleafContext(Request request) {
+    ThymeleafModule.PlayThymeleafContext context = playThymeleafContextFactory.create(request);
+    context.setVariable("tailwindStylesheet", assetsFinder.path("stylesheets/tailwind.css"));
+    context.setVariable("uswdsStylesheet", assetsFinder.path("dist/uswds.min.css"));
+    context.setVariable("adminJsBundle", assetsFinder.path("dist/admin.bundle.js"));
+    context.setVariable("uswdsJsInit", assetsFinder.path("javascripts/uswds/uswds-init.min.js"));
+    context.setVariable("uswdsJsBundle", assetsFinder.path("dist/uswds.bundle.js"));
+    context.setVariable("ApiDocsController", controllers.api.routes.ApiDocsController);
```

Comment:
Is this being used in the template? Should we consider developing some linting to catch unused context variables?

Reply:
nope it wasn't used. I haven't looked into any linting solutions, but that's a good point